plugins/arDominionB5Plugin/modules/right/templates/editSuccess.php

```diff
@@ -185,4 +185,89 @@
 
   </form>
 
+  <script type="text/javascript">
+    (function(){
+
+      // Basis drop-down.
+
+      let BasisSelect = {
+        fieldsets: {
+          copyright: jQuery('#wrapper form div[class *= form-row-copyright]'),
+          license: jQuery('#wrapper div[class *= form-row-license]'),
+          statute: jQuery('#wrapper div[class *= form-row-statute]'),
```

Comment:
Since we need to "duplicate" this JS, instead of adding those classes to the fields I'd try using a different selector ... `.form-control[name^=license]` (you could use the id attribute instead of the name and it may be `.form-check-input` if they're check boxes/radio buttons).

Reply:
Use arrow functions and `$` when moving it to its own JS.